vars/RedHat.yml

```diff
@@ -1,3 +1,4 @@
 ---
 security_ssh_config_path: /etc/ssh/sshd_config
 security_sshd_name: sshd
+security_sshd_handlers: restart ssh
```

Comment:
Do you know if it's not also required on RedHat? It seems like it would be safe to default to it for both Linux flavors.

Reply:
It's not required on RedHat, there's no mention of it in the available docs. It doesn't seem they're planning to go that way - I tested Fedora 41 and was able to change the port without reloading systemd.

But sure, it's sefe to default to it for both, I'll remove the additional variable.